Makefile

```diff
@@ -251,7 +251,13 @@ COVERDB_SUFFIX ?=
 # We use JSON::PP because there is a bug producing a (harmless) 'redefined'
 # warning when using Devel::Cover and Cpanel::JSON::XS
 # https://progress.opensuse.org/issues/90371
-COVEROPT ?= -mJSON::PP -MDevel::Cover=-select_re,'^/lib',+ignore_re,lib/perlcritic/Perl/Critic/Policy,-coverage,statement,-db,cover_db$(COVERDB_SUFFIX),
+#
```

Comment:
should this removed?

Reply:
I added it to separate it from the JSON::PP comment